### ISSUE TITLE
Make calls on $translateProvider chainable

### ISSUE THREAD

**Issue by pmoelgaard** [angular-translate/angular-translate#1041]
It would be neat if it would be possible to chain calls to $translateProvider

**Comment by knalli** [comment]
What do you mean exactly? Because it should be everything chainable...

**Comment by pmoelgaard** [comment]
Currently I have to do the following in the configuration:
```javascript
      $translateProvider.registerAvailableLanguageKeys(['en'], {
        'en_US': 'en',
        'en_UK': 'en'
      });
      $translateProvider.useUrlLoader('i18n/locale')
      $translateProvider.useSanitizeValueStrategy('escaped');
      $translateProvider.useLocalStorage();
      $translateProvider.determinePreferredLanguage();
      $translateProvider.fallbackLanguage('en');
```

However, it would be nice to be able to do the following:
```javascript
      $translateProvider
        .registerAvailableLanguageKeys(['en'], {
          'en_US': 'en',
          'en_UK': 'en'
        })
        .useUrlLoader('i18n/locale')
        .useSanitizeValueStrategy('escaped')
        .useLocalStorage()
        .determinePreferredLanguage()
        .fallbackLanguage('en');
```

**Comment by DWand** [comment]
Hello, @pmoelgaard. It is already chainable.

**Comment by pmoelgaard** [comment]
Well, it's not consistently implemented, I found the first to fail is storageKey(string).

**Comment by pmoelgaard** [comment]
Of the list below, only storageKey seems to return undefined

```javascript

      $translateProvider

        .registerAvailableLanguageKeys(['en'], {
          'en_US': 'en',
          'en_UK': 'en'
        })

        .useUrlLoader('i18n/locale')

        .useSanitizeValueStrategy('escaped')

        .determinePreferredLanguage()
        .fallbackLanguage('en')

        .useLocalStorage()
        .storagePrefix('agilis.inms.')
        // storageKey has to be the last because it's not chained like the others.
        .storageKey('currentLocale');

```

**Comment by pmoelgaard** [comment]
If I'm not mistaken, it seems to be easily solved by adding the following to line 844 in angular-translate.js !

```javascript

return this;

```

**Comment by knalli** [comment]
Fixed in 2.7.2 (released two days ago).
